src/org/usfirst/frc/team1076/robot/subsystems/LeftRightMotors.java

```diff
@@ -20,24 +21,24 @@
 	 */
 	public LeftRightMotors(SpeedController leftMotor, SpeedController rightMotor) {
 		this.leftMotor = leftMotor;
-		this.rightMotor = rightMotor;		
+		this.rightMotor = rightMotor;
 	}
 
 	/**
 	 * Sets the speed of the left motor.
 	 * @param speed in the range -1 to 1 inclusive.
 	 */
 	public void setLeftSpeed(double speed) {
-		this.leftMotor.set(speed);
+		// This motor is backwards
+		this.leftMotor.set(-speed * leftFactor);
```

Comment:
If you look at the main robot class, you'll notice that we adjust it based on SmartDashboard values.